cmsplugin_cascade/apps.py

```diff
@@ -14,7 +14,8 @@ class CascadeConfig(AppConfig):
 
     def ready(self):
         stylesSet = str(settings.CKEDITOR_SETTINGS.get('stylesSet'))
-        if stylesSet != 'default:{}'.format(reverse('admin:cascade_texteditor_config')):
+        if stylesSet != 'default:{}'.format('/admin/cmsplugin_cascade/texteditorconfigfields/wysiwig-config.js',):
```

Comment:
what is the advantage of hard-coding the URL. If a project uses a different path for `/admin/`, then the above wouldn't work anymore.